apps/wallet/src/ui/app/helpers/getMoveTxnSummary.ts

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { getMoveCallTransaction } from '@mysten/sui.js';
+
+import type { SuiTransactionKind } from '@mysten/sui.js';
+
+const stakingCalls = [
+    'request_add_delegation',
+    'request_add_stake_with_locked_coin',
+    'request_withdraw_stake',
+    'request_add_delegation_with_locked_coin',
+    'request_withdraw_delegation',
+    'request_switch_delegation',
+];
+
+// Get known native move function
+export function getMoveCallMeta(txDetails: SuiTransactionKind): {
```

Comment:
I don't really know how this will be used, but this doesn't really seem that generic.

Reply:
On the transaction receipt page, gets the information needed, like the validator address and if it is `Staked` or `Unstaked`...
![Screenshot 2022-12-14 at 7 19 40 PM](https://user-images.githubusercontent.com/8676844/207743912-c83ad16c-fca6-4096-9254-428136af35bd.png)

Reply:
Yes I know, and every single time I say the exact same thing in response that we should find a better way to do this generically instead of hard-coding this copy like this.

Reply:
What suggestion do you have in mind? For example, I am not sure how we can get the validator address meta information from the transaction data as is

Reply:
Looking up validator address metadata probably requires looking at the delegation object created then going through validators in the system object.

My suggestion has long been that we should just display the move call information (e.g. function called), and then work out a scalable solution for more granular copy in parallel (potentially a cross-functional effort with involvement from other engineering teams).